dissect/xfs/xfs.py

```diff
@@ -270,7 +270,6 @@ def filetype(self):
     def link(self):
         if self.filetype != stat.S_IFLNK:
             raise NotASymlinkError(f"{self!r} is not a symlink")
-
```

Comment:
This whitespace removal seems unnecessary

Reply:
OK